src/wp-includes/class-wp-block.php

```diff
@@ -140,6 +140,29 @@ public function __construct( $block, $available_context = array(), $registry = n
 
 		$this->available_context = $available_context;
 
+		$this->refresh_context_dependents();
+		$this->refresh_parsed_block_dependents();
```

Comment:
Are both calls necessary? `refresh_context_dependents` calls `refresh_parsed_block_dependents`, too.

Reply:
Updated in https://github.com/WordPress/wordpress-develop/pull/7522/commits/0b8f5f5355b4f5adc9542cbf15e11d79971f6cfc